dark/views/tournament/team/member/join.py

```diff
@@ -0,0 +1,18 @@
+from django.contrib import messages
+from django.http import HttpResponseRedirect
+from django.urls import reverse
+from django.views.generic.base import View
+
+from dark.models.tournament.team import Team, TeamRole, TeamMember
+
+
+class JoinTeamMemberView(View):
+    def get(self, request, tournament, team):
+        team_obj = Team.objects.get(id=team)
+        user = request.user
+        role = TeamRole.objects.create(team=team_obj, name="Default", can_modify_members=True, can_remove=True)
```

Comment:
chyba powinniśmy to tworzenie roli przenieść do momentu tworzenia zespołu a tutaj tylko szukamy (nie tworzymy) roli o nazwie default (stąd zgaduje został usunięty ten Meta unique_together)

Reply:
Tak jest, usunąłem unique_together bo wywałił mnie błąd. To rozwiązanie, które proponowałeś brzmi bardzo sensownie, poprawię to :)